README.md

```diff
@@ -19,20 +19,30 @@ Once initialized, you don't need to have  `all-contributors-cli` installed globa
 ```console
 npm install --save-dev all-contributors-cli
 ```
+```console
```

Comment:
I'm not sure about adding yarn examples, as it make the documentation bloated in my opinion.
People who use yarn know how to convert a npm command, don't they? 🤔